ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -276,6 +276,24 @@ To retain the old configuration with no TCAM profile the default platform settin
 +      non_mlag: 1020
 ```
 
+### Default value for OSPF area has been updated to `0.0.0.0`
+
+With AVD version 5.0.0, the default value for OSPF area has been updated to "0.0.0.0" to align with EOS's interpretation of the value "0".
+
+```diff
+ethernet_interfaces:
+  - name: Ethernet3.10
+    peer_type: l3_interface
+    ip_address: 10.0.1.1/29
+    shutdown: false
+    description: C1_L3_SERVICE
+    encapsulation_dot1q:
+      vlan: 10
+    vrf: C1_VRF1
+-    ospf_area: '0'
++    ospf_area: 0.0.0.0
+```
```

Comment:
This should show how to revert the change by setting the various vars where we have changed the default.

Reply:
Fixed